src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -197,69 +198,78 @@ const AdminDashboard = (props) => {
     `,
   }
 
-// console.log(UNPAID_SESSIONS)
+  const arr = [];
+
+  const dummy = {
+    course: {startTime: "09:00:00", endTime: "11:00:00", hourlyTuition: 100,},
+    lastPaidSessionDatetime: "2020-06-12T22:15:00+00:00",
+    paymentList: [
+      {
+        parent: {
+          user: {
+            firstName: "Stop", lastName:"Nesting", id: "4",
+            __typename: "ParentType"
+          }
+        }
+      }
+    ],
+  }
+
+  const janetmann = {
+    course: {startTime: "15:15:00", endTime: "16:15:00", hourlyTuition: 100,},
+    lastPaidSessionDatetime: "2020-06-14T22:15:00+00:00",
+    paymentList: [
+      {
+        parent: {
+          user: {
+            firstName: "Janet", lastName:"Mann", id: "3",
+          }
+        }
+      }
+    ],
+  }
 
- const unpaidSessionsData = useQuery(QUERIES["unpaidsessions"])
-//  console.log(unpaidSessionsData)
+
+
+
+ const {data, loading, error} = useQuery(QUERIES["unpaidsessions"])
+//  console.log(data)
+ 
  
 const getTime = (time) => {
```

Comment:
What type of time is this? This is a bit confusing. UTC time? Pacific time? minutes? use a more descriptive variable name